test/MVector.jl

```diff
@@ -48,6 +48,17 @@
         test_expand_error(:(@MVector [i*j for i in 1:2, j in 2:3]))
         test_expand_error(:(@MVector Float32[i*j for i in 1:2, j in 2:3]))
         test_expand_error(:(@MVector [1; 2; 3]...))
+        test_expand_error(:(@MVector a))
+        test_expand_error(:(@MVector [[1 2];[3 4]]))
+
+        if VERSION >= v"1.7.0"
+            @test ((@MVector Float64[1;2;3;;;])::MVector{3}).data === (1.0, 2.0, 3.0)
+            @test ((@MVector [1;2;3;;;])::MVector{3}).data === (1, 2, 3)
+            # @test ((@MVector [;])::MVector{0}).data === ()
```

Comment:
This syntax is not available in 1.7 so maybe it's better to not leave this comment here? I guess it could be in a separate `if VERSION >=` block.

Reply:
Right, even `@static if` won't help here.